Fix #7973: Fixes tutorial, navbar for small screens

ankita240796 opened the pull request:
## Explanation
Fixes #7973 
- Fixes tutorial by removing it for mobile screens (Issue created to add the tutorial for mobile screens: #8043)
- Updates top nav bar to have hamburger on top left and save and publish buttons in correct position for mobile screens. 

## Checklist
- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The linter/Karma presubmit checks have passed.
  - These should run automatically, but if not, you can manually trigger them locally using `python -m scripts.pre_commit_linter` and `python -m scripts.run_frontend_tests`.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR has an appropriate "PROJECT: ..." label (Please add this label for the first-pass review of the PR).
- [x] The PR has an appropriate "CHANGELOG: ..." label (If you are unsure of which label to add, ask the reviewers for guidance).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide).
- [x] The PR addresses the points mentioned in the codeowner checks for the files/folders changed. (See the [codeowner's wiki page](https://github.com/oppia/oppia/wiki/Oppia%27s-code-owners-and-checks-to-be-carried-out-by-developers).)
- [x] The PR is **assigned** to an appropriate reviewer.
  - If you're a new contributor, please ask on [Gitter](https://gitter.im/oppia/oppia-chat) for someone to assign a reviewer and don't tick this checkbox.
  - If you're not sure who the appropriate reviewer is, please assign to the issue's "owner" -- see the "talk-to" label on the issue. Do not only request the review but also add the reviewer as an assignee.

ankita240796 committed:
Fix nav bar for small screens

seanlip commented on core/templates/dev/head/css/oppia.css:
```diff
@@ -2604,7 +2604,7 @@ div#ng-curtain {
 
 .oppia-navbar-brand-name {
-  float: left;
+  float: right;
```
Just to check -- on desktop this still looks good right?

ankita240796 replied:
Yes, it does.

<img width="1678" alt="Screenshot 2019-11-20 at 2 11 32 AM" src="https://user-images.githubusercontent.com/15226041/69184601-2fbb4700-0b3b-11ea-9f7c-f952d634d1ee.png">

seanlip reviewed (approved):
Thanks @ankita240796! No real concerns from my end, just wanted to check one thing.

seanlip commented:
@aks681 PTAL?

aks681 reviewed (approved):
LGTM.

kevintab95 reviewed (approved):
LGTM. Thanks!

kevinlee12 commented:
@ankita240796 e2e tests are broken, ptal!

ankita240796 committed:
Fix tests

ankita240796 committed:
Merge remote-tracking branch 'upstream/develop' into fix-help

ankita240796 commented:
Just a heads up - I have fixed the tests by adding separate protractor classes for small screens. Further, I changed to style to hide save/publish and Oppia text only on "xs" screens (that covers mobiles and when browser is in the minimal size). Text was required to be hidden for xs screens since with the text present, the "User profile" dropdown and "Options" dropdown were overlapping on phones like iPhone 5.

seanlip reviewed (approved):
LGTM from code perspective, thanks Ankita!

kevinlee12 commented:
@nithusha21 ptal!

nithusha21 reviewed (approved):
LGTM, thanks!
